### PR TITLE
Colw/network selection

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
add some nice interaction / animation 😄

File: src/components/common/AppMenu.vue [hunk 1]
```diff
@@ -68,6 +68,20 @@
...
+      <router-link
+        v-if="session.experimentalMode"
+        class="app-menu-item hide-xs"
+        to="/network"
+        exact="exact"
+        title="Network"
+        @click.native="close"
+      >
+        <h2 class="app-menu-title">
+          Network
```
Comment (jbibla) [inline src/components/common/AppMenu.vue]: i think this should all be plural

`networks`

File: src/components/network/NetworkItem.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="network-item">
```
Comment (jbibla) [inline src/components/network/NetworkItem.vue]: reminder: we discussed some indicator for which network is selected. checkmark or pink outline would suffice.

File: src/components/network/PageNetwork.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
+<template>
+  <TmPage data-title="Network" class="small" hide-header>
+    <template>
+      <h3>Mainnets</h3>
+      <ul>
+        <li
+          v-for="network in mainNets"
```
Comment (jbibla) [inline src/components/network/PageNetwork.vue]: this 🐫-casing is so fugly

lol

**Comment by jbibla** [comment]
amazing colin. this is really exciting!

File: src/components/common/AppMenu.vue (hunk 1 above)
Comment (colw) [reply src/components/common/AppMenu.vue]: Done

File: src/components/network/PageNetwork.vue (hunk 3 above)
Comment (colw) [reply src/components/network/PageNetwork.vue]: Haha. I expanded it to `mainNetworks`, which does look a little nicer.

File: src/components/network/NetworkItem.vue (hunk 2 above)
Comment (faboweb) [reply src/components/network/NetworkItem.vue]: Done

**Pull request merged**